### PR TITLE
Add Basic Node Var Type Analysis

### PR DESCRIPTION
**Pull request by zhhsplendid**
1. Move AstNodeWrapper, StaticAnalysisVisitor to a new python file: static_analysis.py
2. Add basic node var type analysis

### CODE REVIEW

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Comment by liym27** [comment]
LGTM

File: python/requirements.txt [hunk 1]
```diff
@@ -18,3 +18,4 @@ decorator
 prettytable
 objgraph
 gast
```
Comment (liym27) [inline python/requirements.txt:20]: gast>=0.3.3 ?

Comment (zhhsplendid) [reply python/requirements.txt:20]: Sure, but since it is minor issue and my CI is waiting, I will change in next PR and inform you.

**Review by liym27 — APPROVED** [review]

**Review by luotao1 — APPROVED** [review]

**Pull request merged**